test-client/src/main/java/io/strimzi/test_client/HttpClientsListener.java

```diff
@@ -45,6 +45,9 @@ public void start() {
                 case DELETE:
                     deleteHandler(request);
                     break;
+                default:
+                    LOGGER.warn("Unexpected HTTP method.");
```

Comment:
You might as well log the `request.method()`, so if it happens you at least know what method is being used.